_data/language-families.yml

```diff
@@ -1,6 +1,7 @@
 ---
 aav: Austroasiatic
 afa: Afro-Asiatic
+aym: Aymaran
```

Comment:
"Aymaran" is for the family, AFAIK, but the language itself is called "Aymara".

Like "Germanic" vs "German".